packages/edit-site/src/components/sidebar/global-styles/palette.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalItemGroup as ItemGroup,
+	FlexItem,
+	__experimentalHStack as HStack,
+	__experimentalZStack as ZStack,
+	__experimentalVStack as VStack,
```

Comment:
It feels like there's too many experimental UI components now, we should start making decisions about stability soon.

Reply:
What do you have in mind? Is this the first place we are using them heavily? Maybe once we are done with the design tools and it all fits we can stabilize.

Reply:
> Maybe once we are done with the design tools and it all fits we can stabilize.

Yes, that makes sense. It will allow us to assert that they are good enough before making the switch. I think some of them are fairly safe like "ZStack, HStack, VStack" though maybe we can be more explicit `HorizontaStack...`

Reply:
I agree with Matias. Let's first use them in a real-world application before deciding if they're stable enough to be marked as non-experimental, especially given how much we want to avoid introducing breaking changes.